Examples/Tutorials/Tutorial-DpdkL2Fwd/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
-add_executable(Tutorial-DpdkL2Fwd main.cpp WorkerThread.cpp)
+cmake_minimum_required(VERSION 3.12)
+project(Tutorial-DpdkL2Fwd)
 
-target_link_libraries(Tutorial-DpdkL2Fwd PUBLIC PcapPlusPlus::Pcap++)
+add_executable("${PROJECT_NAME}" main.cpp)
 
-set_target_properties(Tutorial-DpdkL2Fwd PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PCAPPP_BINARY_TUTORIAL_DIR}")
+if(NOT TARGET PcapPlusPlus::Pcap++)
+        message("Compiling Tutorial out of project")
+        find_package(PcapPlusPlus REQUIRED)
+        # We want to have the binary compiled in the same folder as the .cpp to be near the pcap file
+        set_target_properties("${PROJECT_NAME}" PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
+else()
+        set_target_properties("${PROJECT_NAME}" PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PCAPPP_BINARY_TUTORIAL_DIR}")
```

Comment:
Building the DPDK tutorial as part of the project doesn't work, here the error I got:
```shell
/usr/bin/ld: CMakeFiles/Tutorial-DpdkL2Fwd.dir/main.cpp.o: in function `main':
main.cpp:(.text.startup+0x1eb): undefined reference to `L2FwdWorkerThread::L2FwdWorkerThread(pcpp::DpdkDevice*, pcpp::DpdkDevice*)'
/usr/bin/ld: main.cpp:(.text.startup+0x21b): undefined reference to `L2FwdWorkerThread::L2FwdWorkerThread(pcpp::DpdkDevice*, pcpp::DpdkDevice*)'
collect2: error: ld returned 1 exit status
make[2]: *** [Examples/Tutorials/Tutorial-DpdkL2Fwd/CMakeFiles/Tutorial-DpdkL2Fwd.dir/build.make:113: tutorials_bin/Tutorial-DpdkL2Fwd] Error 1
make[1]: *** [CMakeFiles/Makefile2:2339: Examples/Tutorials/Tutorial-DpdkL2Fwd/CMakeFiles/Tutorial-DpdkL2Fwd.dir/all] Error 2
```

Reply:
should be fixed

Reply:
I can confirm it's fixed, thanks!